commands/resource.js

```diff
@@ -2,7 +2,7 @@ const { YZEmbed } = require('../utils/embeds');
 const YZRoll = require('../yearzero/YZRoll');
 const { emojifyRoll } = require('../Sebedius');
 
-const ARTIFACT_DIE_REGEX = /^d(6|8|10|12)$/i;
+const ARTIFACT_DIE_REGEX = /^[dw](6|8|10|12)$/i;
```

Comment:
Why this change ?

Reply:
This is because i noticed the most common mistake while playing with germans is, that they are used to do common dice rolls as i.e. 1w6, where w stands for the german word for dice "Würfel". I wanted to make it easier for my players :D

Reply:
I reverted the commit